include/blas_meta.h

```diff
@@ -184,6 +184,9 @@ struct is_sycl_scalar
 template <>
 struct is_sycl_scalar<cl::sycl::half> : std::true_type {};
 
+template <>
+struct is_sycl_scalar<float *> : std::false_type {};
+
```

Comment:
Thanks for your PR, @muhammad-tanvir-1211.

What about the case of doubles?

 ```suggestion
template <>
struct is_sycl_scalar<double *> : std::false_type {};
```

Reply:
Fixed this.